src/controllers/auth/login.js

```diff
@@ -0,0 +1,11 @@
+export const login = (res) => {
+  try {
+    return res.status(200).send({
+      message: 'Logging in...',
+    });
+  } catch (error) {
+    return res.status(400).send({
+      error: error.message,
+    });
```

Comment:
@davydocsurg we have an errorHandler express middleware, so instead of short-circuiting the chain of methods in this requests, you want to use the `next` method: https://expressjs.com/en/guide/writing-middleware.html

Reply:
@davydocsurg just want to call out this comment again